### PR TITLE
Rename HCO alerts

### PR DESCRIPTION
**Pull request by machadovilaca**
Follows https://github.com/kubevirt/hyperconverged-cluster-operator/pull/2444 changes

* KubevirtHCOSingleStackIPv6 -> **SingleStackIPv6NotSupported**
* KubevirtHyperconvergedClusterOperatorCRModification -> **KubeVirtCRModified**
* KubevirtHyperconvergedClusterOperatorInstallationNotCompletedAlert -> **HCOInstallationNotCompleted**
* KubevirtHyperconvergedClusterOperatorUSModification -> **UnsupportedHCOModification**

---

* [Removed] KubevirtHyperconvergedClusterOperatorNMOInUseAlert

### CODE REVIEW

**Comment by sradco** [comment]
@apinnick Please review

**Comment by apinnick** [comment]
Might be a good idea to do the same for other acronyms in runbooks like "REST", "HPP", "API", "SSP", "CDI". 

"Hco" is more readable than "HCO" IMO.

**Comment by machadovilaca** [comment]
@apinnick agreed, me and @sradco had that exact discussion on how `KubevirtHcoCRModification`, `KubevirtHcoUSModification` is much more readable than `KubevirtHCOCRModification`, `KubevirtHCOUSModification`

wdyt @sradco? should we change the others?

**Comment by sradco** [comment]
I would like to keep this fix contained. It affects our users. 
If a user silenced the alert based on its name it will now show up again if we change the name.

**Comment by apinnick** [comment]
Also, what about KubevirtHCOSingleStackIPv6?

File: docs/runbooks/README.md [hunk 1]
```diff
@@ -31,10 +31,9 @@ Examples of runbooks can be found in this repository and in the [prometheus-oper
 * [KubeVirtVMIExcessiveMigrations](KubeVirtVMIExcessiveMigrations.md)
 * [KubemacpoolDown](KubemacpoolDown.md)
 * [KubevirtHCOSingleStackIPv6](KubevirtHCOSingleStackIPv6.md)
```
Comment (sradco) [inline docs/runbooks/README.md:33]: @machadovilaca please also update KubevirtHCOSingleStackIPv6

Comment (machadovilaca) [reply docs/runbooks/README.md]: updated

File: docs/runbooks/KubeVirtHcoSingleStackIPv6.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# KubevirtHCOSingleStackIPv6
+# KubeVirtHCOSingleStackIPv6
```
[the remark this answers is not in the document]
Comment (machadovilaca) [reply docs/runbooks/KubeVirtHcoSingleStackIPv6.md]: was just noticing that

**Pull request merged**